proto/backup.proto

```diff
@@ -124,6 +125,26 @@ message S3 {
     string secret_access_key = 9;
 }
 
+// GCS storage backend saves files into google cloud storage.
+message GCS {
+    string endpoint = 1;
```

Comment:
Do we need to specify the `endpoint` when using GCS? S3 storage backend needs this because there are S3 compatible storages.

Reply:
gcs client has this ability, see https://godoc.org/google.golang.org/api/option#WithEndpoint
Maybe useful when users need specify a IP to upload.